install_linux.sh

```diff
@@ -64,6 +65,30 @@ fi
 echo -e "\n\n===================================================="
 echo "Unpacking ${download_zip} ..."
 unzip -u "${download_zip}" -d "${download_path}"
+
+if [ "${SKIP_VERIFY_CHECKSUM}" != "y" ] && [ "${SKIP_VERIFY_CHECKSUM}" != "yes" ]; then
+  echo "Verifying checksum with sha256sum..."
+  set +e
+
+  cd "${download_path}"
+  curl -s -L -o "${checksum_file}" "https://github.com/terraform-linters/tflint/releases/download/${version}/checksums.txt"
+  sha256sum --ignore-missing -c "${checksum_file}" >/dev/null 2>&1
+
+  retVal=$?
+  if [ $retVal -eq 127 ]; then
```

Comment:
Use `command -v` to detect the existence of a program rather than exit code 127

Reply:
I'm also not sure about the logic of implicitly skipping checksum validation when `sha256sum` isn't present. If that's a concern, I'd be inclined to leave checksumming disabled by default and tell users that if they enable it they must provide `sha256sum` to enable checksum verification.